tap_gsheets/__init__.py

```diff
@@ -42,6 +43,11 @@ def sync(config):
         else:
             worksheets = []
 
+        if "date_format" in sheet:
+            gsheet_loader.date_format = sheet["date_format"]
+        else:
+            gsheet_loader.date_format = ""
```

Comment:
What about this suggestion:

```suggestion
        gsheet_loader.date_format = ""
        if "date_format" in sheet:
            gsheet_loader.date_format = sheet["date_format"]
```

Reply:
we can delete the second condition 'else' 
because default value of this var it's none 
https://github.com/miroapp/tap-gsheets/blob/add_date_type_processing/tap_gsheets/gsheet_loader.py#L10